internal/pkg/manifest/pipeline.go

```diff
@@ -190,6 +197,9 @@ func NewPipelineManifest(pipelineName string, provider Provider, stages []Pipeli
 			ProviderName: provider.Name(),
 			Properties:   provider.Properties(),
 		},
+		Build: &Build{
+			Image: DefaultImage,
+		},
```

Comment:
If leave the default to be `nil`, this allow us to move the constant to the deploy pkg and the translation logic to `deploy.PipelineBuildFromManifest`